spec/factories/projects.rb

```diff
@@ -53,4 +53,12 @@
 
     initialize_with { Tmuxinator::Project.new(file) }
   end
+
+  factory :nameless_window_project, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/nameless_window.yml")}")) }
```

Comment:
Line is too long. [95/80]<br>Prefer single-quoted strings inside interpolations.